tests/test_helper.tcl

```diff
@@ -588,7 +588,7 @@ proc print_help_screen {} {
         "--single <unit>    Just execute the specified unit (see next option). This option can be repeated."
         "--verbose          Increases verbosity."
         "--list-tests       List all the available test units."
-        "--only <test>      Just execute tests that match <test> regexp. This option can be repeated."
+        "--only <test>      Just execute the specified test by test name or tests that match <test> regexp. This option can be repeated."
```

Comment:
i think the help must mention that regex must start with a `/`
same should be done for all the skip options, as well as the help in instances.tcl

Reply:
fixed in `bf599dc558f99ad96ef253c1bdd4d530a14b0b17`